contracts/token/airdrop/ArcadeAirdropBase.sol

```diff
@@ -18,8 +18,11 @@ import { AA_ClaimingNotExpired, AA_ClaimingExpired, AA_ZeroAddress } from "../er
  * The contract is ownable, where the owner can reclaim any remaining tokens once the airdrop is
  * over and also change the merkle root and its expiration at their discretion.
  */
-contract ArcadeAirdrop is ArcadeMerkleRewards, Authorizable {
+abstract contract ArcadeAirdropBase is ArcadeMerkleRewards, Authorizable {
```

Comment:
What is the motivation for separating `ArcadeAirdropBase` and `ArcadeMerkleRewards`? I'm not sure what other contracts would inherit `ArcadeMerkleRewards`